### PR TITLE
Portfolio : mobile-version

### PR DESCRIPTION
**Pull request by arnoldnekemiah**
- Configured Linters into the project.
- Followed the recommended github flow.
- Update Readme file to describe the project.
- Added Toolbar and Headline sections.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div class="logo">
+          <a href="" id="logo">Ikonde Arnold Nekemiah</a>
+        </div>
+        <nav id="navi" class="dropdown">
+            <img class="menu" id="btn-menu" src="./Images/Icon - Menu.png" alt="menu">
+        </nav>
```
Comment (Olamarx) [inline index.html]: - [x] This is a width of 376px and I could not see your hamburger, I advise you to use flexbox to style it.
image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  color: #fff;
+  text-decoration: none;
+}
+
+/* headline section */
+.headline {
+  background: #1c1a19;
+  color: #fff;
+  padding: 130px 24px 145px;
+  background-image: url(./Images/Group.png);
```
Comment (Olamarx) [inline style.css:30]: - [x] You will notice that this image came out well in the figma design, kindly adapt it to yours also.

## Figma
image

## Yours
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @arnoldnekemiah, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️ 
- Readme file professional ✔️

## Required Changes ♻️
_Check the comments under the review._
- [ ] On a general note, kindly use the appropriate font as used in the design. For the `Hey There. I am Anold`, you will notice it has a different font family.

### Figma

image

### Yours
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+.headline {
+  background: #1c1a19;
+  color: #fff;
+  padding: 130px 25px 145px;
+  background-image: url(./Images/Group.png);
+  background-position: 294px 380px;
+  background-repeat: no-repeat;
+  width: 376px;
+  margin: auto;
+}
```
Comment (levy002) [inline style.css]: - [ ] As requested by the previous reviewer, the hamburger icon and background image must be visible on the required minimum mobile width of `375px` but I realize in order to see everything on your page we need to scroll horizontally. Kindly make sure to remove this error which is causing the overflow horizontally. so your website contents must all be correctly visible on a minimum width of `375`px without scrolling.

  - ### YOUR APP BEHAVIOUR ON WIDTH OF `375PX`
  
  chrome-capture-2023-3-27 (4)

  - ### FIGMA DESIGN ON WIDTH OF `375PX`
  image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @arnoldnekemiah   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @arnoldnekemiah, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 

## Optional suggestions
- [ ] [OPTIONAL] Between `375px` and `400px`, it is still not possible to see the whole project. It would be nice if you correct it 🙏🏼 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**